test/python/expected/upgrade_validation/expected_dependency.out

```diff
@@ -507,6 +525,7 @@ Function sys.pgerror()
 Function sys.pltsql_call_handler()
 Function sys.pltsql_inline_handler(internal)
 Function sys.pltsql_validator(oid)
+Function sys.point(sys.geometry)
```

Comment:
All new entries in this file mean there are no upgrade tests using those functions. Please add proper upgrade tests, if possible.

Reply:
I have added tests for most of the functions just there are some Cast functions which need some modifications in receiver part, which will be addressed in upcoming PR.